src/Routes/Api.ts

```diff
@@ -30,5 +34,14 @@ export function createApiRoutes(
         [validator.body(CreateDiskInforamtionValidationSchema), passNodeData],
         diskInformationController.createDiskData.bind(diskInformationController));
 
+    router.post(
+        "/node/uptime",
+        [validator.body(CreateNodeUptimeValidationSchema), passNodeData],
+        nodeUptimeController.storeNodeUptime.bind(nodeUptimeController));
+
+    router.get(
+        "/node/uptime",
+        passNodeData,
```

Comment:
This is not a good idea actually. GET requests should have no body. However, in this case we can use ID as we will have all user's nodes on frontend and we have the control over the hosted app (in case of reporting it's different). So, the route would be `/node/uptime/:id`